### PR TITLE
New question: Does my ISP know if I'm using Tor?

### PR DESCRIPTION
**Pull request by enjeck**
Fixing this issue: https://dip.torproject.org/torproject/web/community/issues/91

### CODE REVIEW

File: content/tbb/why-isp-not-block-tor/contents.lr [hunk 1]
```diff
@@ -0,0 +1,19 @@
+_model: question
+---
+title: Why doesn't the ISP just block Tor?
+---
+seo_slug: why-isp-not-block-tor
+---
+description:
+### Does my ISP know if I'm using Tor?
+Your Internet Service Provider (ISP), may be able to see that you're using Tor, but they won't know where you're going when you do.
+You can prevent your ISP from knowing this by using a VPN.
```
Comment (gusgustavo) [inline content/tbb/why-isp-not-block-tor/contents.lr]: You can prevent your ISP from knowing this by using a Bridge.

File: content/tbb/why-isp-not-block-tor/contents.lr [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+---
+title: Why doesn't the ISP just block Tor?
+---
+seo_slug: why-isp-not-block-tor
+---
+description:
+### Does my ISP know if I'm using Tor?
+Your Internet Service Provider (ISP), may be able to see that you're using Tor, but they won't know where you're going when you do.
+You can prevent your ISP from knowing this by using a VPN. 
+Using a VPN with Tor is not recommended, so you should consider using [bridges](https://tb-manual.torproject.org/bridges/) instead.
```
Comment (gusgustavo) [inline content/tbb/why-isp-not-block-tor/contents.lr]: We don't need a VPN to make a private access to the tor network.

File: content/tbb/why-isp-not-block-tor/contents.lr [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+seo_slug: why-isp-not-block-tor
+---
+description:
+### Does my ISP know if I'm using Tor?
+Your Internet Service Provider (ISP), may be able to see that you're using Tor, but they won't know where you're going when you do.
+You can prevent your ISP from knowing this by using a VPN. 
+Using a VPN with Tor is not recommended, so you should consider using [bridges](https://tb-manual.torproject.org/bridges/) instead. 
+
+### Can Tor be blocked?
+Yes, some governments or network admistrators may be able to block Tor. They can get the list of Tor relays and block them.
```
Comment (gusgustavo) [inline content/tbb/why-isp-not-block-tor/contents.lr]: We should say that the list of Tor relays is public.

File: content/tbb/why-isp-not-block-tor/contents.lr [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+### If a company bans you from using Tor, what can you do to continue using?
```
Comment (gusgustavo) [inline content/tbb/why-isp-not-block-tor/contents.lr]: We should replace "company" with something more generic.

File: content/tbb/why-isp-not-block-tor/contents.lr [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+You can bypass this by using [bridges](https://tb-manual.torproject.org/bridges/).
```
Comment (gusgustavo) [inline content/tbb/why-isp-not-block-tor/contents.lr:16]: I'm merging Circumvention Tools and Pluggable Transports page. 
The new one will be /circumvention.

**Review by gusgustavo — CHANGES_REQUESTED** [review]
Good work! There are some changes to be made :)

**Comment by enjeck** [comment]
@gusgustavo Hi. What do you think of the changes?

**Pull request closed without merging**